src/mlpack/methods/det/det_main.cpp

```diff
@@ -61,6 +66,22 @@ PARAM_MATRIX_OUT("test_set_estimates", "The output estimates on the test set "
 PARAM_MATRIX_OUT("vi", "The output variable importance values for each "
     "feature.", "i");
 
+// Tagging and path printing options
+PARAM_STRING_IN("path_format", "The format of path printing - lr|id-lr|lr-id",
+                "p", "lr");
+
+PARAM_STRING_OUT("tag_counters_file", "The file to output number of points "
+                 "that went to each leaf.", "c");
+
+PARAM_STRING_OUT("raw_estimates_file", "The file to output the estimations from"
+                 " the unpruned tree.", "u");
+
+PARAM_STRING_OUT("tag_file", "The file to output the tags (and possibly paths)"
+                 " for each sample in the test set.", "g");
+
+PARAM_FLAG("skip_pruning", "Whether to bypass the pruning process and output "
+              "the unpruned tree only", "s");
```

Comment:
Can you add a period at the end of this comment for consistency?  Also, it seems like using `-s -e file.csv` is equivalent to `-u file.csv`.  Do you think maybe we should remove the `--raw_estimates_file` option then?

Reply:
Yes, indeed. The `skip-pruning` functionality is more generic. A tiny inconvenience would be that if you want to have _both_ raw and final estimates, you'll need to run it twice.